webui/react/src/components/ProjectMoveModal.tsx

```diff
@@ -45,13 +45,23 @@ const ProjectMoveModalComponent: React.FC<Props> = ({ onMove, project }: Props)
       });
       onMove?.();
     } catch (e) {
-      handleError(e, {
-        level: ErrorLevel.Error,
-        publicMessage: 'Please try again later.',
-        publicSubject: 'Unable to move project.',
-        silent: false,
-        type: ErrorType.Server,
-      });
+      if (e instanceof DetError && e.type === ErrorType.Server) {
```

Comment:
only difference here is `publicMessage`, so i think we can use ternary operator.

Reply:
Updated